### PR TITLE
(vi): Update some translations and spelling mistake in Vietnamese

### PR DESCRIPTION
**Pull request by xcaro**
## Description
Update some translations and spelling mistake in Vietnamese

### CODE REVIEW

**Comment by minimalsm** [comment]
Thanks for this @xcaro :-) just getting this double-checked and we'll get it merged ❤️

**Comment by xcaro** [comment]
> Thanks for this @xcaro :-) just getting this double-checked and we'll get it merged ❤️

hi @minimalsm, what I need to do wth this pr?

File: src/intl/vi/common.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
   "consensus-run-beacon-chain": "Chạy một máy khách đồng thuận",
   "consensus-run-beacon-chain-desc": "Ethereum cần nhiều clients hoạt động nhất có thể. Sử dụng một dịch vụ chung của Ethereum này!",
   "consensus-when-shipping": "Khi nào đi vào hoạt động?",
-  "eth-upgrade-what-happened": "Chuyện gì xảy ra vởi 'Eth2'?",
-  "eth-upgrade-what-happened-description": "Thuật ngữ 'Eth2' đã không được dùng nữa khi chúng ta tiếp cập giai đoạn hợp nhất. 'Lớp đồng thuận' chính là 'Eth2' trước đây.",
+  "eth-upgrade-what-happened": "Chuyện gì xảy ra với 'Eth2?'",
+  "eth-upgrade-what-happened-description": "The term 'Eth2' has been deprecated as we approach the merge. The 'consensus layer' encapsulates what was formerly known as 'Eth2.'",
```
Comment (minimalsm) [inline src/intl/vi/common.json:63]:
```suggestion
  "eth-upgrade-what-happened-description": "Thuật ngữ 'Eth2' đã không được dùng nữa khi chúng ta tiếp cập giai đoạn hợp nhất. 'Lớp đồng thuận' chính là 'Eth2' trước đây.",
```

**Comment by minimalsm** [comment]
@xcaro merged 😀

**Pull request merged**